public/css/list/comment-list.less

```diff
@@ -47,3 +47,14 @@
     white-space: normal;
   }
 }
+
+.controls .comment-list .list-item {
+  .visual {
+    width: auto;
+
+    .user-ball {
+      // Align optically to `quick-actions`
+      margin-left: -1px;
```

Comment:
Which quick actions?? There are none in a comment's detail/details view.

Also, the negative margin feels arbitrary if applied to the user-ball. Apply it to the visual if you absolutely have to.

Reply:
✅